src/toast/tests/timing.py

```diff
@@ -68,19 +68,13 @@ def test_global(self):
         prec = 1
         gt = GlobalTimers.get()
         tnames = ["timer1", "timer2", "timer3"]
-        for nm in tnames:
-            try:
-                gt.stop(nm)
-                tm.report("This should raise since timer not started...")
-            except:
-                print("Successful exception:  stop an already stopped timer")
         for nm in tnames:
             gt.start(nm)
         for nm in tnames:
             self.assertTrue(gt.is_running(nm))
             try:
                 s = gt.seconds(nm)
-            except:
+            except Exception:
```

Comment:
This would look more professional if we caught the subclass of Exception that we anticipate

Reply:
Changed to RuntimeError, which is what is thrown from the C++ code.